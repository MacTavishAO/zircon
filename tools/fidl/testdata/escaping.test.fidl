library escapeme;

/// "pleaseescapethisdocommentproperly"
struct DocCommentWithQuotes {
};
