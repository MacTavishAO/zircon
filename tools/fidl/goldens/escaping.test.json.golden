{
  "version": "0.0.1",
  "name": "escapeme",
  "library_dependencies": [],
  "bits_declarations": [],
  "const_declarations": [],
  "enum_declarations": [],
  "experimental_resource_declarations": [],
  "interface_declarations": [],
  "service_declarations": [],
  "struct_declarations": [
    {
      "name": "escapeme/DocCommentWithQuotes",
      "location": {
        "filename": "escaping.test.fidl",
        "line": 4,
        "column": 8,
        "length": 20
      },
      "anonymous": false,
      "maybe_attributes": [
        {
          "name": "Doc",
          "value": " \"pleaseescapethisdocommentproperly\"\n"
        }
      ],
      "members": [],
      "type_shape_v1": {
        "inline_size": 1,
        "alignment": 1,
        "depth": 0,
        "max_handles": 0,
        "max_out_of_line": 0,
        "has_padding": false,
        "has_flexible_envelope": false,
        "is_resource": false
      }
    }
  ],
  "table_declarations": [],
  "union_declarations": [],
  "type_alias_declarations": [],
  "declaration_order": [
    "escapeme/DocCommentWithQuotes"
  ],
  "declarations": {
    "escapeme/DocCommentWithQuotes": "struct"
  }
}
